hibike/hibike_process.py

```diff
@@ -14,41 +15,18 @@
 uid_to_index = {}
 
 def hibike_process(badThingsQueue, stateQueue, pipeFromChild):
-    
-    # Last command is included so that it's compatible with OS X Sierra
-    # Note: If you are running OS X Sierra, do not access the directory through vagrant ssh
-    # Instead access it through Volumes/vagrant/PieCentral
-    ports = glob.glob("/dev/ttyACM*") + glob.glob("/dev/ttyUSB*") + glob.glob("/dev/tty.usbmodem*")
 
-    try:
-        virtual_device_config_file = os.path.join(os.path.dirname(__file__), "virtual_devices.txt")
-        ports.extend(open(virtual_device_config_file, "r").read().split())
-    except IOError:
-        pass
-
-    serials = []
-
-    working_ports = []
-
-    for port in ports:
-        try:
-            serials.append(serial.Serial(port, 115200))
-            working_ports.append(port)
-
-            #this implimentation ensures that as long as the cannot open serial port error occurs while opening the serial port, a print will appear, but the rest of the ports will go on.
-
-        except serial.serialutil.SerialException as e:
-            print("Cannot Open Serial Port: " + str(port))
+    devices = hid.open_all()
```

Comment:
Well then. Shouldn't that mean that this commit also fixes virtual devices so they work again?

(Pottery Barn Rule)